targets/TARGET_ublox/TARGET_R5030/device/cmsis_nvic.h

```diff
@@ -18,7 +18,12 @@
 #ifndef MBED_CMSIS_NVIC_H
 #define MBED_CMSIS_NVIC_H
 
-#define NVIC_NUM_VECTORS      (16 + 32)   // CORE + MCU Peripherals
+
+// CORE: 16 vectors = 64 bytes from 0x00 to 0x3F
+// MCU Peripherals: 45 vectors =  180 bytes from 0x40 to 0x6D
```

Comment:
I make 180 bytes to be from 0x40 to 0xF4.

Reply:
Corrected. Also, there are 62 vectors in total now.